roles/conjur_host_identity/tasks/identity.yml

```diff
@@ -44,6 +44,11 @@
     mode: 0644
 
 - block:
+  - name: Warn against disabling cert validation
+    debug:
```

Comment:
Does this mean that the warning will only show up if debug logging is turned on? If so, is there a higher level that could be used so the warning is always shown?

Reply:
The `debug` module has a [`verbosity` parameter](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/debug_module.html#parameters) that controls this:

> A number that controls when the debug is run, if you set to 3 it will only run debug when -vvv or above.
>
> Default: 0

Given that the default is `0`, this message should always display if the `when` condition is satisfied.